### ISSUE TITLE
Which version of FF should we support for a11y?

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/a11y-research#102]
~7 months ago, FF released "Firefox Quantum" (version 57), which caused substantial problems for a11y. FF and AT vendors recommended that users use Firefox 52 ESR until problems were fixed.

Since then FF and AT vendors have been working on improving performance and fixing bugs introduced in Quantum. As of FF 60, JAWS should theoretically be usable (thanks @lmulhall-phet for finding this)

> JAWS 2018 now includes improvements necessary for users who move to Mozilla FireFox version 60, expected to release within the month. It is advised that JAWS users remain on the ESR version of FireFox until Mozilla officially releases version 60.

But when I try latest version of JAWS (18.0.5xxx), I see the following dialog and I cannot use the AT at all:
![capture](https://user-images.githubusercontent.com/6396244/40142663-7ee8a7d8-5916-11e8-8110-8e6f2892d5f9.PNG)

NVDA is usable on FF 60, but is substantially slower than FF 52 ESR.

On the other hand, we are finding some bugs that are in FF 52 ESR that are *not* found in FF 60, like https://github.com/phetsims/balloons-and-static-electricity/issues/405.

Which version of FF should we support?

In the long run, we should support latest version of FF. As of january this year, 2/3 of FF users on NVDA are using latest version (https://www.marcozehe.de/2018/01/16/nvda-firefox-58-team-regaining-strength/)

But if FF 57+ performance is too poor to use a sim, do we spend resources on optimization when it is 
likely that FF will be significantly faster for a11y in ~6 months? And FF 60 isn't even usable (!!) with JAWS (at least for me, @lmulhall-phet had better luck on his machine).

It would be good to discuss with @terracoda @emily-phet @mbarlow12 and @zepumph in the next a11y meeting.

**Comment by terracoda** [comment]
@jessegreenberg, thanks for posting this issue! I am sorry I will miss the a11y meeting this Friday, but I will try to catch up on this topic.

**Comment by terracoda** [comment]
And here is another article on FF 60 and JAWS from Marco on May 11, 2018:
[https://www.marcozehe.de/2018/05/11/firefox-60-and-jaws-2018-back-in-good-browsing-conditions-together/](https://www.marcozehe.de/2018/05/11/firefox-60-and-jaws-2018-back-in-good-browsing-conditions-together/)

**Comment by jessegreenberg** [comment]
Awesome, thanks for posting that @terracoda. It sounds like FF is announcing "Mission Accomplished", which indicates to me that we should be striving or FF latest.

From the article:
> The version you should be using with Firefox 60 is at least 2018.1804.26.

I wasn't able to get this version from the "Check for updates" option in the help menu. But the version listed is available for download from the website. Ill try that and see if I am able to use FF again.

**Comment by jessegreenberg** [comment]
That version of JAWS sounds nice on FF 60, it works! That really helps decide, thanks @terracoda.

**Comment by emily-phet** [comment]
@jessegreenberg Keeping with latest FF sounds correct to me. We'll just have to make the call on specific issues on a case-by-case basis (as you mentioned).
